server/utils.js

```diff
@@ -31,6 +31,14 @@ var FOOD_ESTABLISHMENT_TYPE = 'FoodEstablishment';
 var DATE_TYPE = 'Date';
 var AGGREGATE_RATING_TYPE = 'AggregateRating';
 
+/**
+ * Function used in restaurantfeeder.js to download the json restaurant data
+ *
+ * @param {Object} options - Options of the request
+ * @param {Function} callback - Callback function
+ * @param {Object} res - Response object
+ * @param {Boolean} useHttps - Boolean to use https
+*/
 function doGet(options, callback, res, useHttps) {
   var protocol = http;
   if (useHttps) {
```

Comment:
Function `doGet` was used in the old feeders, will be removed in an upcoming PR!